app/views/shared/fields/_relationships_data.html.erb

```diff
@@ -4,9 +4,9 @@
 %>
 
 	<ul class="unstyled glyphicon-list glyphicon-list-tag">
-		<% value.sort.each do |name| %>
-			<li><%= link_to name, facet_uri(name, 'creator_sim') %></li>
+		<% value.each do |name| %>
```

Comment:
Are there any reasons for not sorting names?

Reply:
@lsitu - oh nice, i think there was an issue w/ relationship names and not having a sort_by method, perhaps the class was different or something. I'll double check.

Reply:
@lsitu - thanks for catching that :) I must have just missed this one, and am thinking of a different file that I couldn't apply `sort_by` to. Just pushed an amended commit.